json/wireguard.json

```diff
@@ -30,5 +30,14 @@
         "username": "admin",
         "password": "admin"
     },
-    "notes": []
-}
\ No newline at end of file
+  "notes": [
+    {
+      "text": "WGDashboard and Wireguard are not the same thing! Issues with Wireguard do not relate to WGDashboard.",
+      "type": "info"
+    },
+    {
+      "text": "You can find an Example-Configuration here: nano /etc/wireguard/wg0.conf",
```

Comment:
```suggestion
```
Think we should remove this since all wireguard configurations are stored in `/etc/wireguard` and the example config is explicitly mentioned in install process and it will be visible in WGDashboard. Also some people might create their own `wg0.conf`, then this could cause confusion

Reply:
fine by me.